### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by vflorin1995**
### Changes

- Add basic UI
- Add Main class

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+require_relative './app'
+
+def main
+  app = Main.new
+  app.menu
+end
+
+main
```
Comment (BenMukebo) [inline main.rb]: - [ ] The `main.rb` file is the entry point that is invoked at the end of the file. You should Present the user with a list of options to perform here and not in the app.js file.
- This method should do and look similar to the following:

```ruby
def main
  app = App.new()

  response = nil

  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    when "3"
      app.create_person()
       ...
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

main()
```

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    true
+  end
+end
+
+class Main
+  def initialize
+    @book = []
+    @people = []
+    @rental = []
+  end
```
Comment (BenMukebo) [inline app.rb]: - [ ] This should be `class App` and not the `class Main` 🤔

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+  def list_all_rentals
+    if @rental.empty?
+      puts("\n\nThere are no rentals\n\n")
+    else
+      puts 'please type person id'
+      id = gets.chomp.to_i
+      @rental.each do |r|
+        puts "\n\n#{r.book} rented by #{r.person} on #{r.date}\n\n" if r.person.id == id
+      end
+    end
```
Comment (BenMukebo) [inline app.rb]: - [ ] The school library console app should behave in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided for you. But the ` list_all_rentals` option does not work as expected, this is the screenshot 👇🏻 The book title and author does not show correctly, Kindly fix this 👍🏻 



- Your console:

Screenshot from 2022-12-22 18-14-43


- Video provided:

Screenshot (1)

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @vflorin1995,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @vflorin1995, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**